packages/react-router/package.json

```diff
@@ -0,0 +1,102 @@
+{
+  "name": "@clerk/react-router",
+  "version": "0.0.0",
+  "description": "Clerk SDK for React Router",
+  "keywords": [
+    "clerk",
+    "typescript",
+    "react-router",
+    "auth",
+    "authentication",
+    "passwordless",
+    "session",
+    "jwt"
+  ],
+  "homepage": "https://clerk.com/",
+  "bugs": {
+    "url": "https://github.com/clerk/javascript/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/clerk/javascript.git",
+    "directory": "packages/react-router"
+  },
+  "license": "MIT",
+  "author": "Clerk",
+  "sideEffects": false,
+  "type": "module",
```

Comment:
ESM-only!

Reply:
React Router is Vite-based now, so you mostly see official and unofficial examples using Vitest. So without having any hard numbers I assume most people use Vitest, not Jest. Thus I wouldn't care too much about that limitation, in Jest-land you can workaround it, too.